core/services/ocr2/plugins/ocr2keeper/custom_telemetry.go

```diff
@@ -0,0 +1,127 @@
+package ocr2keeper
+
+import (
+	"context"
+	"time"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+	"google.golang.org/protobuf/proto"
+
+	httypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker/types"
+	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/synchronization/telem"
+	"github.com/smartcontractkit/chainlink/v2/core/static"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+const (
+	customTelemChanSize = 100
+)
+
+type AutomationCustomTelemetryService struct {
+	utils.StartStopOnce
+	monitoringEndpoint commontypes.MonitoringEndpoint
+	headBroadcaster    httypes.HeadBroadcaster
+	headCh             chan blockKey
+	unsubscribe        func()
+	chDone             chan struct{}
+	lggr               logger.Logger
```

Comment:
add a field for config digest and initialize it in `NewAutomationCustomTelemetryService`